src/app/pages/confirm-test-details/confirm-test-details.page.html

```diff
@@ -1,11 +1,86 @@
 <ion-header>
+  <practice-mode-banner *ngIf="isPracticeMode"></practice-mode-banner>
   <ion-toolbar>
-    <ion-title>confirm-test-details</ion-title>
+    <ion-title>Confirm test details - {{pageState.candidateUntitledName$ | async}}</ion-title>
   </ion-toolbar>
+  <ion-buttons id="back-button" slot="start">
+    <ion-back-button></ion-back-button>
+  </ion-buttons>
 </ion-header>
 
-<ion-content>
-  <ion-button (click)="navigateBack()">Back</ion-button>
-  <ion-button (click)="navigateDebrief()">To Debrief</ion-button>
-  <ion-button (click)="navigateBackToOffice()">To Back to Office</ion-button>
+<ion-content class="ion-padding" force-overscroll="false">
```

Comment:
`force-overscroll="false"` is attached to ion-content here, but in DES3 its equivalent (`no-bounce`) is attached to the `ion-card-content`

Not sure if this makes a difference but just pointing it out!

Reply:
yes this is deliberate. the equivalent to `no-bounce` can only be attached to `ion-content` in ionic 5 and since it would be the content that you are scrolling it makes sense.